.github/workflows/build_and_test.yaml

```diff
@@ -6,6 +6,8 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
+      - name: Checkout
```

Comment:
I don't think we need this here - any specific reason for it?

Reply:
btw: it would be fine for me if we'd simply duplicate the png file in the server folder. Or use a symbolic link within the repository.
I think the docker build action builds everything via buildx and hence remotely and therefore clones the repo within the build action itself/

Reply:
We could also just move the png into the server folder and refer to it from the assertion scripts.

Reply:
I'll give it a last try, seems that only the static/ dir is missing (had it locally but forgot git does not push empty dirs), 

> I don't think we need this here - any specific reason for it?
buildx by default runs on a fresh git checkout, so you can't modify the build context with previous steps (here adding the png image).
Checking out and specifying the context path explicitly allows to circumvent this limitation.